commands/apps_dev.go

```diff
@@ -516,8 +516,8 @@ func appDevPrepareEnvironment(ctx context.Context, ws *workspace.AppDev, cli bui
 		}
 
 		// TODO: get stack run image from builder image md after we pull it, see below
-		images = append(images, "digitaloceanapps/apps-run:heroku-18_db5978a")
-		images = append(images, "digitaloceanapps/apps-run:heroku-22_db5978a")
+		images = append(images, "digitaloceanapps/apps-run:heroku-18_df7e351")
+		images = append(images, "digitaloceanapps/apps-run:heroku-22_df7e351")
```

Comment:
Looking at the available tags for this image, even the new ones are four months old. Should we consider using a new one? Is there a changelog for the image? 

https://hub.docker.com/r/digitaloceanapps/apps-run/tags

Reply:
Actually, the `apps-run` image is pinned with `cnb-local-image` https://github.com/digitalocean/doctl/blob/32566d9ea889b07dd9cb1e2e07b537633028a688/internal/apps/builder/cnb.go#L29

i.e. `cnb-local-image v0.65.0` uses `apps-run image: df7e351`.

The newer tags are for unreleased dev versions (which are still unstable).

Reply:
:+1:  Thanks for the additional info!